projects/novo-elements/src/elements/data-table/data-table.component.ts

```diff
@@ -358,7 +359,7 @@ export class NovoDataTable<T> implements AfterContentInit, OnDestroy {
       // Re-subscribe
       this.outsideFilterSubscription = outsideFilter.subscribe((filter: any) => {
         this.state.outsideFilter = filter;
-        this.state.updates.next({ globalSearch: this.state.globalSearch, filter: this.state.filter, sort: this.state.sort, where: this.state.where });
+        this.state.updates.next({ globalSearch: this.state.globalSearch, filter: this.state.filter, sort: this.state.sort, where: this.state.where, savedSearchName: this.state.savedSearchName });
```

Comment:
Not sure the DataTable needs to know about the savedSearch???  Could this not be stored in the custom DataTableState service in your implmentation?

Reply:
i pulled it out of all the `updates` events and the state clear functions. i kept it in the new setState function since that's needed to set the preferences and the preferencesChanged event emitter since that's passing an IDataTablePreferences obj which includes the savedSearchName.